karapace.config.json

```diff
@@ -1,4 +1,5 @@
 {
+    "access_logs_debug": false,
```

Comment:
I'm not sure if this is the best approach. If we follow this style we would end up with a flag for each logger. It makes more sense to add a logging config dictionary:

https://docs.python.org/3/library/logging.config.html#logging.config.dictConfig

And disable the logging for that particular logger, partial example:

```json
{
  "logging_config": {
    "version": 1,
    "loggers": {
      "aiohttp.access": {
        "level": "CRITICAL"
      }
    }
  }
}
```

That needs additional code though, to setup the handlers.

Reply:
Humm, I forgot about some of the rationale here. The idea was exactly to patch aiohttp to change the log level, because it is easier to configure https://github.com/aiven/journalpump to filter messages by that. I suppose the alternative would be to have a search with a negative REGEX, but that may not be the best idea. So never mind my suggestion above.